packages/flutter_tools/lib/src/build_info.dart

```diff
@@ -776,6 +783,55 @@ AndroidArch getAndroidArchForName(String platform) {
   throw Exception('Unsupported Android arch name "$platform"');
 }
 
+String getNameForAndroidArch(AndroidArch arch) {
```

Comment:
This section seems unexpected - perhaps this was a bad merge conflict?

Reply:
You're right, that was a bad merge, sorry.